packages/tpetra/core/src/Tpetra_CrsGraph_def.hpp

```diff
@@ -4735,6 +4752,70 @@ namespace Tpetra {
     } // if my process doesn't have local constants
   }
 
+  template <class LocalOrdinal, class GlobalOrdinal, class Node, const bool classic>
+  void
+  CrsGraph<LocalOrdinal, GlobalOrdinal, Node, classic>::
+  computeTriangularProperties ()
```

Comment:
See above note (on calling this "computeLocalTriangularProperties").  Also, it might make sense for this to be a free function that takes a `local_graph_type` input, instead of an instance method of (the global data structure) `Tpetra::CrsGraph`.  That's because the function only knows how to work on the `local_graph_type` instance, not on any of CrsGraph's other data structure options.